Thai/urbex_bkk

```diff
@@ -0,0 +1,10 @@
+Have you ever explored a deserted building in Bangkok ?
+
+   Sathorn Unique Tower, facing Chao Phraya River, is an abandoned, incomplete 49-storey tower built during 1990s. 
+With its location, many photographers and urbexes pay a visit to this tower in order to find a good spot to shoot chic photos
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.